src/index.native.tsx

```diff
@@ -249,9 +249,17 @@ class Screen extends React.Component<ScreenProps> {
         activityState,
         children,
         isNativeStack,
+        gestureResponseDistance,
         ...props
       } = rest;
 
+      const defaultedGestureResponseDistance = {
+        start: gestureResponseDistance?.start ?? -1,
+        end: gestureResponseDistance?.end ?? -1,
+        top: gestureResponseDistance?.top ?? -1,
+        bottom: gestureResponseDistance?.bottom ?? -1,
+      };
```

Comment:
This could be inlined to avoid creating an unnecessary variable. Otherwise LGTM

Reply:
See: [77883f](https://github.com/software-mansion/react-native-screens/pull/1507/commits/77883ff9eec7f0527657bff171492cc6b5ab5e69)